### ISSUE TITLE
kiuserexceptiondispatcher

### ISSUE THREAD

**Issue by d0c59** [x64dbg/x64dbg#3290]
### Operating System

Windows 11 Home 22H2 build 22621.2861

### x64dbg Version

Dec 4, 2023

### Describe the issue

Tried jit debugging.  Won't go thru code. Jumps to ntdll.dll module and freezes on KiUserExceptionDispatcher.  Is there a way around this?

### Steps to reproduce

1. Create int3 PE in flatassembler
2. Run the asm
3. Step over int3 and program freezes

**Comment by mrexodia** [comment]
Did you get it working?

**Comment by d0c59** [comment]
No.  I just switched to a different debugger.  I didn't want to bother you
with this issue that's why I closed it out.
